RetailCoder.VBE/Inspections/QuickFixes/PassParameterByReferenceQuickFix.cs

```diff
@@ -15,11 +15,13 @@ namespace Rubberduck.Inspections.QuickFixes
     public class PassParameterByReferenceQuickFix : QuickFixBase
     {
         private Declaration _target;
+        private AssignedByValParameterQuickFixHelper _quickFixHlpr;
```

Comment:
why is the `AssignedByValParameterQuickFixHelper` used in `PassParameterByReferenceQuickFix`?? This seems incorrect. Probably the best workaround would be to name it `QuickFixHelper`, largely because it seems completely unrelated to any special quick-fix as of now..